caas/kubernetes/provider/k8s_test.go

```diff
@@ -2164,6 +2112,7 @@ password: shhhh`[1:],
 			Annotations: map[string]string{
 				"juju.io/controller": testing.ControllerTag.Id(),
 				"fred":               "mary",
+				"juju-app-uuid":      "appuuid",
```

Comment:
Do we want the app uuid even when there's no storage?

Reply:
Currently, statefulset always has `juju-app-uuid` set in annotations.
Do we want to make it consistent with statefulset?

Reply:
No harm I guess in always having it